spec/integration/legacy_sitemaps_spec.rb

```diff
@@ -35,7 +35,6 @@ def site_with_sitemap(&block)
 
     it 'extracts links from sitemap' do
       expect(results).to have_only_these_results [
-        mock_response(url: 'http://127.0.0.1:9393/', status_code: 404),
```

Comment:
These needed to be removed because the previous version of crawler was outputting errors and redirects as crawl results. We don't want that anymore, so it's expected that these errors (and redirects in other tests) will no longer appear in crawl results.